src/util/strutil.c

```diff
@@ -8,65 +8,132 @@
 #include <ctype.h>
 #include "RG.h"
 #include "rmalloc.h"
-#include "deps/utf8proc/utf8proc.h"
+#include "utf8proc/utf8proc.h"
```

Comment:
I'll assume the folder `deps` is on the include path e.g. compiler flag `-I` ?

Reply:
yes